controller/userController.js

```diff
@@ -170,101 +155,118 @@ const retrieveById = (req, res) => {
             return res.status(HttpStatus.OK).json(user);
         } else {
             return res.status(HttpStatus.NOT_FOUND).json({
-                status: StatusEnum['ERROR'],
                 message: 'User not found.'
             });
         }
     }).catch(err => {
         return res.status(HttpStatus.INTERNAL_SERVER_ERROR).json({
-            status: StatusEnum['ERROR'],
             message: ValidationUtil.buildErrorMessage(err, 'retrieveById', 'user')
         });
     });
 };
 
-const update = (req, res) => {
+const changePersonalData = (req, res) => {
     const loggedUser = req.decoded;
 
     if (!ValidationUtil.isValidObjectId(loggedUser._id)) {
-        return res.status(HttpStatus.BAD_REQUEST).json({
-            status: StatusEnum['ERROR'],
-            message: 'Invalid ID'
-        });
+        return res.status(HttpStatus.BAD_REQUEST).json({ message: 'Invalid ID' });
     }
     user = {};
-    // TODO: 
-    // Make 4 functions from this one:
-    //  1. updateFields() (everything except password & active flag)
-    //  2. updatePassword()
-    //  3. blockUser (basically remove)
-    //  3. activateUser
     // Update fields
     if (req.body.name)
         user.name = req.body.name;
     if (req.body.surname)
         user.surname = req.body.surname;
-    if (req.body.email)
-        user.email = req.body.email;
     if (req.body.dateOfBirth)
         user.dateOfBirth = req.body.dateOfBirth;
```

Comment:
I believe we also have a `profilePic` attribute. Maybe that should be included in the update as well, or perhaps we should wait to see how the integration with AWS will work before plastering a simple `string` here. Maybe a more complex property will be needed.